langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaChatModel.java

```diff
@@ -212,7 +243,7 @@ public OllamaChatModelBuilder stop(List<String> stop) {
             return this;
         }
 
-        public OllamaChatModelBuilder format(String format) {
+        public OllamaChatModelBuilder format(ResponseFormat format) {
             this.format = format;
```

Comment:
This is a breaking change, is there a way to avoid it?

Reply:
Can we still support method that accepts string and make it deprecated?

Reply:
Okay, so I brought back the `format` field as a String (marked as deprecated) and introduced a new field, `responseFormat`.
I made the following assumption: when both fields are used with the builder, I throw an `IllegalStateException`, as only one can be used at a time. This assumption simplifies determining what and when a field is used.

Given this, the old APIs like `generate(messages)` can utilize either the deprecated `format` field or the new `responseFormat` field. However, the new ChatRequest API will always use the `responseFormat` field.
What are your thoughts?

Reply:
@bidek sounds reasonable, thank you!